### PR TITLE
New TRG 1.06 - Administrator's Guide

### PR DESCRIPTION
**Pull request by Siegfriedk**
Please be aware: while I created this PR, this TRG is not from me, i'm only preparing and moderating the merging of the new TRG 1.06.

### CODE REVIEW

**Comment by FaGru3n** [comment]
guess with this little changes we could easily adpot the change

**Comment by Siegfriedk** [comment]
> guess with this little changes we could easily adpot the change, but checks should not failing.

its failing only because of the place holder. the final PR will be working

File: docs/release/trg-1/trg-1-06.md
Comment (SebastianBezold) [inline docs/release/trg-1/trg-1-06.md:1]: "Admin Guide is available with the software at the same time" -> Same as in other TRG drafts, I don't really see what that should mean, so I would remove it
"appropriate maturity" -> what should that mean. Again we also do not hand over anything. we must not introduce a hint of liability here. Anyone can use stuff in Eclipse Tractus-X as a gift,
Last bullet point in the description should be proper sentences. Can maybe be copied from the TRG 1.05 proposal

in best-practice:
"updated, once an up-revision is foreseen" -> what is "up-revision"?? Why not just say, it is up-to-date 
appropriate maturity -> again totally unclear what that should mean and how to verify.
Delete second bullet point

**Review by SebastianBezold — DISMISSED** [review]

File: docs/release/trg-1/trg-1-06.md
Comment (Siegfriedk) [reply docs/release/trg-1/trg-1-06.md:1]: @RolaH1t pls advice || @kelaja

**Review by florianrusch-zf — APPROVED** [review]

**Review by mhellmeier — APPROVED** [review]

File: docs/release/trg-1/trg-1-06.md
Comment (Sicaine) [reply docs/release/trg-1/trg-1-06.md:1]: @SebastianBezold pls review. everything adjusted

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**